### DISCUSSION TITLE
Can't seem to enable the Addon Docs Tabs in a new Storybook Project

### DISCUSSION THREAD

**Discussion by nolrinale** [storybookjs/storybook#25188] [Help] [answered]
### Summary

When you visualize a component it is possible to enable a tab in the main toolbar which allows you to switch between the component and it's docs, however creating a brand new storybook project with your Stackblitz template I am unable to reproduce this. 

Is it necessary to enable something in particular to achieve this? 

I can't seem to find proper documentation explaining how to enable this behavior

**Comment by Marklb** [comment, marked as answer]
Are you referring to:
![image](https://github.com/storybookjs/storybook/assets/8348647/5fb2448f-04bc-4c01-8148-11e29a2a6d85)

I don't know if there is an option to restore those buttons, without maybe using the API, but I think those were replaced by [Autodocs](https://storybook.js.org/docs/writing-docs/autodocs). That docs tab went to a page that was for all the stories in the `*.stories.(ts|mdx)` file, not docs for just that story. Now, there is a Docs item in the sidenav to go to that page: 
![image](https://github.com/storybookjs/storybook/assets/8348647/8ecee7a7-65db-49e0-a44c-765c51f870d5)

Someone else can probably explain the reason for the change, better than me, or may know where a discussion on it is.

**Reply by nolrinale** [reply]
Oh! I didn't know they were deliberately removed in the latest SB version, that makes things easier then, I will just convince the bosses that it just doesn't exists anymore. 

Thanks for the explanation!
